estate/models/estate_property.py

```diff
@@ -0,0 +1,68 @@
+from odoo import api,fields, models
+# import for adding months with relativedelta
+from dateutil.relativedelta import relativedelta 
+
+
+class PropertyPlan(models.Model):
+
+    #function calculting total area by adding living area and garden area
+    @api.depends('living_area', 'garden_area')
+    def _compute_total_area(self):
+        for record in self:
+            record.total_area = record.living_area + record.garden_area
+
+    #best price calculation
+    @api.depends('offer_ids.price')
+    def _computer_best_price(self):
+        for record in self:
+            if record.offer_ids:
+                record.best_price = max(record.offer_ids.mapped('price'))
+            else:
+                record.best_price = 0.0
+
+    _name = "estate.property"
+    _description = "Estate property tables"
```

Comment:
```suggestion
    _name = "estate.property"
    _description = "Estate property tables"

    #function calculting total area by adding living area and garden area
    @api.depends('living_area', 'garden_area')
    def _compute_total_area(self):
        for record in self:
            record.total_area = record.living_area + record.garden_area

    #best price calculation
    @api.depends('offer_ids.price')
    def _computer_best_price(self):
        for record in self:
            if record.offer_ids:
                record.best_price = max(record.offer_ids.mapped('price'))
            else:
                record.best_price = 0.0
```

The model class declaration should be followed by the name and description.
Additionally, the description should clearly explain the purpose of adding the model.